qt/aqt/sound.py

```diff
@@ -711,9 +711,10 @@ def setup_audio(taskman: TaskManager, base_folder: str) -> None:
 
         av_player.players.append(MacTTSPlayer(taskman))
     elif isWin:
-        from aqt.tts import WindowsTTSPlayer
+        from aqt.tts import WindowsTTSPlayer, WindowsRTTTSFilePlayer
```

Comment:
Please add a check for platform.release() == "10" - if it's not 10, we should avoid importing and adding the new code, which should avoid breaking older clients.